### DISCUSSION TITLE
server 的-interface如何使用

### DISCUSSION THREAD

**Discussion by AbyssShang** [mokeyish/smartdns-rs#242] [Q&A]
server 的-interface如何使用？为何我设置之后报错？
![image](https://github.com/mokeyish/smartdns-rs/assets/110171545/94eed3b6-fcad-4aa3-90b8-56d1862010dd)
报错信息
![image](https://github.com/mokeyish/smartdns-rs/assets/110171545/d6ece504-0904-4dc2-aea0-1a9c1348d3f7)

**Comment by mokeyish** [comment]
应该是配置解析的 bug，我找个时间修复一下。

**Reply by AbyssShang** [reply]
順便問一下，該如何設置代理？我現在使用clash，但是開系統代理之後，好像就優先走clash的dns

**Reply by mokeyish** [reply]
你可以为上游服务器，配置 socks5 代理。

**Comment by mokeyish** [comment]
interface 的已经支持了。
